### PR TITLE
docs(accordion): update accordion code page with react usage

### PR DESCRIPTION
**Pull request by joshblack**
Closes https://github.com/carbon-design-system/carbon/issues/6006

Add accordion documentation for React in the code tab.

#### Changelog

**New**

**Changed**

- Update Accordion code docs with React usage docs

**Removed**

### CODE REVIEW

File: src/pages/components/accordion/code.mdx [hunk 1]
```diff
@@ -1,36 +1,201 @@
...
+`AccordionItem` components. The `Accordion` components accepts a list of
+`AccordionItem` components as `children` and each `AccordionItem` is responsible
+for displaying the accordion's heading and panel content.
+
+You can configure the accordion item's heading using the `title` prop.
+Everything you pass in as a child of `AccordionItem` will be rendered in the
+accordion's panel.
+
+```jsx
+import { Accordion, AccordionItem } from "carbon-components-react";
```
Comment (joshblack) [inline src/pages/components/accordion/code.mdx:36]: @vpicone any tips on getting prettier to format these with single quotes? 👀

Comment (vpicone) [reply src/pages/components/accordion/code.mdx:36]: Sure, what for?

Comment (vpicone) [reply src/pages/components/accordion/code.mdx:36]: Just for this one snippet?

Comment (joshblack) [reply src/pages/components/accordion/code.mdx:36]: For codeblocks in `mdx` files, prettier will format them using the default configuration for the language. The default configuration for prettier in JSX files uses double quotes and we would like to use single quotes.

**Pull request merged**